doc/source/management/data_at_rest_encryption.rst

```diff
@@ -437,7 +437,12 @@ System variables
    :vartype: Boolean
    :default: ``Off``
 
-Enables the encryption of InnoDB Undo tablespaces
+Enables the encryption of InnoDB Undo tablespaces.
+
+.. note::
+
+    After you have encrypted the undo log, you cannot disable the undo log encryption.
```

Comment:
You can disable it, and if you do, newly written undo pages won't be encrypted. The setting is dynamic, you can turn it on and off while the server is running.

The limitation is that the encryption info stays in the header, and the server will try to load it during startup. If it can't do it (e.g. no keyring plugin is loaded), it can't start.